<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton, #pushButton_6{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}

QPushButton#pushButton:hover, #pushButton_6:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButton:pressed, #pushButton_6:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}

QPushButton#pushButton_2, #pushButton_3, #pushButton_4, #pushButton_5{
	background-color: rgba(0, 0, 0, 0);
	color:rgba(85, 98, 112, 255);
}

QPushButton#pushButton_2:hover, #pushButton_3:hover, #pushButton_4:hover, #pushButton_5:hover{
	color: rgba(131, 96, 53, 255);
}

QPushButton#pushButton_2:pressed, #pushButton_3:pressed, #pushButton_4:pressed, #pushButton_5:pressed{
	padding-left:5px;
	padding-top:5px;
	color:rgba(91, 88, 53, 255);
}

</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>280</width>
        <height>430</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(2, 65, 118, 255);
color:rgba(255,255,255,200);
border-radius:5px;
border-top-left-radius: 50px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>0</y>
        <width>240</width>
        <height>430</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(255, 255, 255, 255);
border-bottom-right-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>230</width>
        <height>130</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0, 0, 0, 75);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>231</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgba(255, 255, 255, 200);</string>
      </property>
      <property name="text">
       <string>TO DO LIST</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe Script</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgba(255, 255, 255, 170);</string>
      </property>
      <property name="text">
       <string>Gününüzü rahatça planlayın!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>220</width>
        <height>315</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>221</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(0, 0, 0, 200);</string>
       </property>
       <property name="text">
        <string>GİRİŞ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_eposta_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>  E-posta</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sifre_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101, 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>  Şifre</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_giris">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>201</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton_giris {
    background-color: rgba(2, 65, 118, 255);
    color: rgba(255, 255, 255, 200);
    border-radius: 5px;
}

QPushButton#pushButton_giris:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgba(2, 65, 118, 100);
    background-position: calc(100% - 10px) center;
}</string>
       </property>
       <property name="text">
        <string>Giriş</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_kayitol">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>280</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton_kayitol {
    background-color: white;
    color: black;
    border-radius: 5px;
}

QPushButton#pushButton_kayitol:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgba(2, 65, 118, 100);
    background-position: calc(100% - 10px) center;
}

QPushButton#pushButton_kayitol:hover {
    color: rgba(2, 65, 118, 255); /* İlk arka plan için kullanılan koyu mavi renk */
}</string>
       </property>
       <property name="text">
        <string>Kayıt Ol</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#pushButton{
background-color:rgba(2, 65, 118, 255);
color:rgba(255,255,255,200);
border-radius:5px;
}
@PushButton#pushButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,100);
background-position:calc(100% - 10px)center;
}
QLabel#pushButton{
background-color:rgba(2,65,118,200);
}</string>
       </property>
       <property name="text">
        <string>Üye değil misin?</string>
       </property>
      </widget>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_sifremiunuttum">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>390</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>8</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QCommandLinkButton#commandLinkButton_sifremiunuttum {
    background-color: white;
    color: black;
    border: none;
}

QCommandLinkButton#commandLinkButton_sifremiunuttum:pressed {
    padding-left: 5px;
    padding-top: 5px;
    background-color: rgba(2, 65, 118, 100);
    background-position: calc(100% - 10px) center;
}

QCommandLinkButton#commandLinkButton_sifremiunuttum:hover {
    color: rgba(2, 65, 118, 255); /* İlk arka plan için kullanılan koyu mavi renk */
}</string>
      </property>
      <property name="text">
       <string>Şifremi unuttum</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>380</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>6</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgba(255, 255, 255, 170);</string>
      </property>
      <property name="text">
       <string>Acar Software™ © 2024 Tüm hakları saklıdır.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>label_8</zorder>
     <zorder>label_7</zorder>
     <zorder>label_11</zorder>
     <zorder>label_10</zorder>
     <zorder>widget_4</zorder>
     <zorder>commandLinkButton_sifremiunuttum</zorder>
     <zorder>label_12</zorder>
     <zorder>label_9</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
